README.md

```diff
@@ -144,6 +144,12 @@ This will remove everything! (images, containers, volumes)
 
 *Docker stores MySQL data locally in the directory `.data`. If you want to fully clean you'll have to remove this folder.*
 
+## Populating Copyright information in footer
```

Comment:
Added the documentation in the README so people will know to wire this piece when running for their institution